pkgs/development/tools/gertty/default.nix

```diff
@@ -0,0 +1,48 @@
+{ python3, python3Packages, stdenv }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "gertty";
+  version = "1.6.0";
+
+  nativeBuildInputs = with python3Packages; [
+    pbr
+    voluptuous
+    dateutil
+    requests
+    urwid
+    ply
+    ordereddict
+    alembic
+    GitPython
+    pyyaml
+  ];
+
+  # XXX Looks like all the buildInputs needs to be propagated...
+  propagatedBuildInputs = with python3.pkgs; [
+    pbr
+    voluptuous
+    dateutil
+    requests
+    urwid
+    ply
+    ordereddict
+    alembic
+    GitPython
+    pyyaml
+  ];
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "1pvpca59z5rcir7lz2yimwiqjvd8yhnwjskxna2bslyfwv996d8w";
+  };
+
+  doCheck = true;
```

Comment:
The logs show that `Ran 0 tests in 0.000s`. This might mean that the the package either doesn't contain any tests or that `python setup.py tests` of the project is not properly set up. in which case some python packages require manual definition of the  `installCheckPhase`. Could you verify which one is the case?

Reply:
Looks like there are no tests in the repo. The only thing seems to be some style check, which I'm not really performing here then.